tests/manager_socket/utils.py

```diff
@@ -147,18 +147,14 @@ def check_filter_files(filter_name, extension=".1"):
 # Responses
 
 RESP_EMPTY     = '{}'
-RESP_ONE       = '{"logs_1": {}}'
-RESP_THREE     = '{"logs_1": {}, "logs_2": {}, "logs_3": {}}'
-RESP_STATUS_OK = '{"status": "OK"}'
-RESP_STATUS_KO_GEN = '{"status": "KO"'
-RESP_STATUS_KO_NO_PID = '{"status": "KO", "errors": [{"filter": "logs_1", "error": "PID file not accessible"}]}'
-RESP_STATUS_KO_NO_RUN = '{"status": "KO", "errors": [{"filter": "logs_1", "error": "Process not running"}]}'
-RESP_STATUS_KO_NO_MAIN_SOCK = '{"status": "KO", "errors": [{"filter": "logs_1", "error": "Main socket not created"}]}'
-RESP_STATUS_KO_NO_MON_SOCK = '{"status": "KO", "errors": [{"filter": "logs_1", "error": "Monitoring socket not created"}]}'
-RESP_STATUS_KO_NO_MON_SOCK_NREADY = '{"status": "KO", "errors": [{"filter": "logs_1", "error": "Monitoring socket not ready"}]}'
-RESP_STATUS_KO_LIST = [ RESP_STATUS_KO_NO_PID,
-                        RESP_STATUS_KO_NO_RUN,
-                        RESP_STATUS_KO_NO_MAIN_SOCK,
-                        RESP_STATUS_KO_NO_MON_SOCK,
-                        RESP_STATUS_KO_NO_MON_SOCK_NREADY]
-RESP_FILTER_NOT_EXISTING = '{"status": "KO", "errors": [{"filter": "tototititata", "error": "Filter not existing"}]}'
\ No newline at end of file
+RESP_LOGS_1 = '"logs_1":'
+RESP_LOGS_2 = '"logs_2":'
+RESP_LOGS_3 = '"logs_3":'
```

Comment:
Do not test if the filter is running because the field is present even when the filter is down.

When must test the value (a valid json if the filter is running, null otherwise).

```suggestion
RESP_LOGS_3 = '"logs_3": {}'
```

Repeat the suggestion on RESP_LOGS_1 and RESP_LOGS_2.

Reply:
My suggestion might need to be updated according to #78

Reply:
good call !

I'm solving this